modules/ps_banner/ps_banner.tpl

```diff
@@ -4,7 +4,7 @@
  *}
 <a class="banner" href="{$banner_link}" title="{$banner_desc}">
```

Comment:
```suggestion
<a class="d-block" href="{$banner_link}" title="{$banner_desc}">
```

Reply:
Adding `d-block` class is still good. No more css needed.